kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -408,14 +423,13 @@ public void initialize() {
         });
 
         workspace.addEventHandler(MouseEvent.MOUSE_CLICKED, this::onMouseClickedOnDesktopSurfacePane);
-
-        // Refresh Concept navigator
-        refreshCalculatorEventSubscriber = _ -> {
-           // TODO FIXME Must refresh the cache or invalidate cache after a gitsync or import. Below works but will change the UI.
-           // ViewCoordinateHelper.changeViewCalculatorToLatestByTime(getNavigatorNode().getViewProperties(), System.currentTimeMillis() + );
-           getNavigatorNode().getController().refresh();
```

Comment:
@besidev 
The removed code between lines _****411 - 416** should be included**_ and lines **426 - 431** should be removed.

Reply:
@besidev Can you resync from main again and resolve the merge conflict?
I just added another fix in this file.